.github/policies/resourceManagement.yml

```diff
@@ -144,6 +144,28 @@ configuration:
           label: Community
       description: Label Community Pull Requests
       triggerOnOwnActions: false
+    - if:

```

Comment:
Does this actually work? I've not see the equivalent API Owners tagging actually work well.

Reply:
I have no idea. I based it on the other PR taggers, but am not sure there is a good way to test.

Reply:
Looking at [docs](https://eng.ms/docs/more/github-inside-microsoft/policies/examples/pr-triage) and I think the indentation is all wrong. Will update.

Reply:
Doesn't look like the formatting was _that_ different after all. Still made changes to improve readability.